parsers/manifest_parser_test.go

```diff
@@ -893,6 +896,42 @@ func TestComposeActionsForFunctionWithRemoteDir(t *testing.T) {
 	assert.NotNil(t, err, "Compose actions should have exited with error when code is specified but runtime is missing.")
 }
 
+// validate manifest_parser.ComposeActions() method
+func TestComposeActionsForDocker(t *testing.T) {
+
+	file := "../tests/dat/manifest_data_compose_actions_for_docker.yaml"
+	p, m, _ := testLoadParseManifest(t, file)
+
+	actions, err := p.ComposeActionsFromAllPackages(m, m.Filepath, whisk.KeyValue{})
+	assert.Nil(t, err, fmt.Sprintf(TEST_ERROR_COMPOSE_ACTION_FAILURE, file))
+
+	var expectedResult, actualResult string
+	for _, action := range actions {
+		switch action.Action.Name {
+		case "OpenWhiskSkeleton":
+		case "OpenWhiskSkeletonWithNative":
+			assert.Equal(t, utils.BLACKBOX, action.Action.Exec.Kind, fmt.Sprintf(TEST_MSG_ACTION_DOCKER_KIND_MISMATCH, action.Action.Exec.Kind))
+			assert.Equal(t, NATIVE_DOCKER_IMAGE, action.Action.Exec.Image, TEST_MSG_ACTION_DOCKER_IMAGE_MISMATCH)
+		case "CustomDockerAction1":
+		case "CustomDockerAction2":
+			expectedResult, _ = filepath.Abs("../tests/src/integration/docker/actions/exec.zip")
```

Comment:
Would have preferred this declared at top of function so we can quickly identify test data files used within the test method and they do not scroll off the right edge.

Reply:
👍 done